static/js/components/auth/AuthEmailForm.js

```diff
@@ -2,49 +2,114 @@
 /* global SETTINGS:false */
 import React from "react"
 import ReCAPTCHA from "react-google-recaptcha"
+import styled from "styled-components"
+import debounce from "lodash/debounce"
 
 import { validationMessage } from "../../lib/validation"
 
 import type { EmailForm } from "../../flow/authTypes"
 import type { FormProps } from "../../flow/formTypes"
 
-const AuthEmailForm = ({
-  form,
-  validation,
-  onSubmit,
-  onUpdate,
-  onRecaptcha,
-  processing
-}: FormProps<EmailForm>) => {
-  return (
-    <form onSubmit={onSubmit} className="form">
-      <div className="emailfield row">
-        <input
-          type="email"
-          name="email"
-          value={form.email}
-          onChange={onUpdate}
-          placeholder="Email"
-        />
-        {validationMessage(validation.email)}
-      </div>
-      {SETTINGS.recaptchaKey && onRecaptcha ? (
-        <div className="recaptcha row">
-          <ReCAPTCHA sitekey={SETTINGS.recaptchaKey} onChange={onRecaptcha} />
-          {validationMessage(validation.recaptcha)}
+type Props = FormProps<EmailForm>
+type State = {
+  recaptchaScale: number
+}
+
+// These needs to stay in sync with ReCAPTCHA's runtime width
```

Comment:
I'm not sure what this comment means - these constants are the width and height the captcha component renders at normally, right? if so, maybe calling them something like `RECAPTCHA_NATURAL_WIDTH` or `RECAPTCHA_INHERENT_WIDTH` or something along those lines would capture what the value represents more clearly.